network/server_discovery.go

```diff
@@ -7,6 +7,7 @@ import (
 	"math/big"
 	"time"
 
+	helperCommon "github.com/dogechain-lab/dogechain/helper/common"
```

Comment:
Why not place those functions in `network/common`?

Reply:
`Max`, `Min`, `Clamp`, these math functions are all too common